service/history/queues/executable.go

```diff
@@ -234,6 +217,25 @@ func (e *executableImpl) Execute() (retErr error) {
 		// Not doing it here as for certain errors latency for the attempt should not be counted
 	}()
 
+	ns, err := e.namespaceRegistry.GetNamespaceByID(namespace.ID(e.GetNamespaceID()))
+	if err != nil {
+		return err
```

Comment:
need to populate `e.taggedMetricsHandler` by doing 
`e.taggedMetricsHandler = e.metricsHandler.WithTags(e.estimateTaskMetricTag()...)`

same for L236

Reply:
Added them